### ISSUE TITLE
Bug: Gluetun keeps resatrting after upgrade from 3.33 to v3.34.2

### ISSUE THREAD

**Issue by capsel22** [qdm12/gluetun#1626]
### Is this urgent?

No

### Host OS

Ubuntu 22.04

### CPU arch

x86_64

### VPN service provider

Mullvad

### What are you using to run the container

Portainer

### What is the version of Gluetun

v3.34.2

### What's the problem 🤔

Today I upgraded from 3.33.0 to newest v3.34.2. The container is in a unhealthy loop, it keeps restarting.
There was no docker-compose change after upgrade, everything stayed the same.
When reverted back to 3.33.0 everything works again.

Attaching v3.34.2 and 3.33.0 logs below. Let me know what else I can supply. Happy to assist.

### Share your logs

```plain text
v3.34.2 logs:
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================

Running version latest built on 2023-05-29T20:47:00.502Z (commit 4759392)

🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-05-30T08:43:58+01:00 INFO [routing] default route found: interface eth0, gateway 172.17.0.1, assigned IP 172.17.0.6 and family v4
2023-05-30T08:43:58+01:00 INFO [routing] local ethernet link found: eth0
2023-05-30T08:43:58+01:00 INFO [routing] local ipnet found: 172.17.0.0/16
2023-05-30T08:43:58+01:00 INFO [firewall] enabling...
2023-05-30T08:43:58+01:00 INFO [firewall] enabled successfully
2023-05-30T08:43:58+01:00 INFO [storage] creating /gluetun/servers.json with 13056 hardcoded servers
2023-05-30T08:43:58+01:00 INFO Alpine version: 3.18.0
2023-05-30T08:43:58+01:00 INFO OpenVPN 2.5 version: 2.5.8
2023-05-30T08:43:58+01:00 INFO OpenVPN 2.6 version: 2.6.4
2023-05-30T08:43:58+01:00 INFO Unbound version: 1.17.1
2023-05-30T08:43:58+01:00 INFO IPtables version: v1.8.9
2023-05-30T08:43:58+01:00 INFO Settings summary:
├── VPN settings:
|   ├── VPN provider settings:
|   |   ├── Name: mullvad
|   |   └── Server selection settings:
|   |       ├── VPN type: wireguard
|   |       ├── Cities: amsterdam
|   |       └── Wireguard selection settings:
|   └── Wireguard settings:
|       ├── Private key: kkb...lu=
|       ├── Interface addresses:
|       |   └── redacted
|       └── Network interface: tun0
|           └── MTU: 1420
├── DNS settings:
|   ├── DNS server address to use: 127.0.0.1
|   ├── Keep existing nameserver(s): no
|   └── DNS over TLS settings:
|       ├── Enabled: yes
|       ├── Update period: every 24h0m0s
|       ├── Unbound settings:
|       |   ├── Authoritative servers:
|       |   |   └── cloudflare
|       |   ├── Caching: yes
|       |   ├── IPv6: no
|       |   ├── Verbosity level: 1
|       |   ├── Verbosity details level: 0
|       |   ├── Validation log level: 0
|       |   ├── System user: root
|       |   └── Allowed networks:
|       └── DNS filtering settings:
|           ├── Block malicious: yes
|           ├── Block ads: no
|           ├── Block surveillance: no
|           └── Blocked IP networks:
|               ├── 127.0.0.1/8
|               ├── 10.0.0.0/8
|               ├── 172.16.0.0/12
|               ├── 192.168.0.0/16
|               ├── 169.254.0.0/16
|               ├── ::1/128
|               ├── fc00::/7
|               ├── fe80::/10
|               ├── ::ffff:127.0.0.1/104
|               ├── ::ffff:10.0.0.0/104
|               ├── ::ffff:169.254.0.0/112
|               ├── ::ffff:172.16.0.0/108
|               └── ::ffff:192.168.0.0/112
├── Firewall settings:
|   ├── Enabled: yes
|   ├── VPN input ports:
|   |   └── 57160
|   └── Outbound subnets:
|       └── 192.168.1.0/24
├── Log settings:
|   └── Log level: INFO
├── Health settings:
|   ├── Server listening address: 127.0.0.1:9999
|   ├── Target address: cloudflare.com:443
|   ├── Duration to wait after success: 5s
|   ├── Read header timeout: 100ms
|   ├── Read timeout: 500ms
|   └── VPN wait durations:
|       ├── Initial duration: 6s
|       └── Additional duration: 5s
├── Shadowsocks server settings:
|   └── Enabled: no
├── HTTP proxy settings:
|   ├── Enabled: yes
|   ├── Listening address: :8888
|   ├── User: 
|   ├── Password: [not set]
|   ├── Stealth mode: no
|   ├── Log: no
|   ├── Read header timeout: 1s
|   └── Read timeout: 3s
├── Control server settings:
|   ├── Listening address: :8000
|   └── Logging: yes
├── OS Alpine settings:
|   ├── Process UID: 1001
|   ├── Process GID: 1001
|   └── Timezone: europe/london
├── Public IP settings:
|   ├── Fetching: every 12h0m0s
|   └── IP file path: /tmp/gluetun/ip
└── Version settings:
    └── Enabled: yes
2023-05-30T08:43:58+01:00 INFO [routing] default route found: interface eth0, gateway 172.17.0.1, assigned IP 172.17.0.6 and family v4
2023-05-30T08:43:58+01:00 INFO [routing] adding route for 0.0.0.0/0
2023-05-30T08:43:58+01:00 INFO [firewall] setting allowed subnets...
2023-05-30T08:43:58+01:00 INFO [routing] default route found: interface eth0, gateway 172.17.0.1, assigned IP 172.17.0.6 and family v4
2023-05-30T08:43:58+01:00 INFO [routing] adding route for 192.168.1.0/24
2023-05-30T08:43:58+01:00 INFO [dns over tls] using plaintext DNS at address 1.1.1.1
2023-05-30T08:43:58+01:00 INFO [http server] http server listening on [::]:8000
2023-05-30T08:43:58+01:00 INFO [http proxy] listening on :8888
2023-05-30T08:43:58+01:00 INFO [healthcheck] listening on 127.0.0.1:9999
2023-05-30T08:43:58+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T08:43:58+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T08:43:58+01:00 INFO [wireguard] Connecting to 185.65.134.222:51820
2023-05-30T08:43:58+01:00 INFO [wireguard] Wireguard is up
2023-05-30T08:43:58+01:00 INFO [firewall] setting allowed input port 57160 through interface tun0...
2023-05-30T08:43:58+01:00 INFO [dns over tls] downloading DNS over TLS cryptographic files
2023-05-30T08:44:06+01:00 INFO [healthcheck] program has been unhealthy for 6s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-05-30T08:44:06+01:00 INFO [vpn] stopping
2023-05-30T08:44:06+01:00 INFO [firewall] removing allowed port 57160...
2023-05-30T08:44:06+01:00 ERROR [vpn] cannot get version information: Get "https://api.github.com/repos/qdm12/gluetun/commits": context canceled
2023-05-30T08:44:06+01:00 INFO [vpn] starting
2023-05-30T08:44:06+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T08:44:06+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T08:44:06+01:00 INFO [wireguard] Connecting to 92.60.40.224:51820
2023-05-30T08:44:06+01:00 INFO [wireguard] Wireguard is up
2023-05-30T08:44:07+01:00 INFO [firewall] setting allowed input port 57160 through interface tun0...
2023-05-30T08:44:08+01:00 WARN [dns over tls] cannot update files: Get "https://www.internic.net/domain/named.root": dial tcp: lookup www.internic.net on 1.1.1.1:53: read udp 10.65.43.151:37031->1.1.1.1:53: i/o timeout
2023-05-30T08:44:08+01:00 INFO [dns over tls] attempting restart in 10s
2023-05-30T08:44:18+01:00 INFO [healthcheck] program has been unhealthy for 11s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-05-30T08:44:18+01:00 INFO [vpn] stopping
2023-05-30T08:44:18+01:00 INFO [firewall] removing allowed port 57160...
2023-05-30T08:44:18+01:00 INFO [vpn] starting
2023-05-30T08:44:18+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T08:44:18+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T08:44:18+01:00 INFO [wireguard] Connecting to 169.150.196.15:51820
2023-05-30T08:44:18+01:00 INFO [wireguard] Wireguard is up
2023-05-30T08:44:18+01:00 INFO [firewall] setting allowed input port 57160 through interface tun0...
2023-05-30T08:44:18+01:00 INFO [dns over tls] downloading DNS over TLS cryptographic files
2023-05-30T08:44:28+01:00 WARN [dns over tls] cannot update files: Get "https://www.internic.net/domain/named.root": dial tcp: lookup www.internic.net on 1.1.1.1:53: read udp 10.65.43.151:57097->1.1.1.1:53: i/o timeout
2023-05-30T08:44:28+01:00 INFO [dns over tls] attempting restart in 20s
2023-05-30T08:44:34+01:00 INFO [healthcheck] program has been unhealthy for 16s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-05-30T08:44:34+01:00 INFO [vpn] stopping
2023-05-30T08:44:34+01:00 INFO [firewall] removing allowed port 57160...
2023-05-30T08:44:34+01:00 INFO [vpn] starting
2023-05-30T08:44:34+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T08:44:34+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T08:44:34+01:00 INFO [wireguard] Connecting to 169.150.196.2:51820
2023-05-30T08:44:34+01:00 INFO [wireguard] Wireguard is up
2023-05-30T08:44:34+01:00 INFO [firewall] setting allowed input port 57160 through interface tun0...
2023-05-30T08:44:48+01:00 INFO [dns over tls] downloading DNS over TLS cryptographic files
2023-05-30T08:44:57+01:00 INFO [healthcheck] program has been unhealthy for 21s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-05-30T08:44:57+01:00 INFO [vpn] stopping
2023-05-30T08:44:57+01:00 INFO [firewall] removing allowed port 57160...
2023-05-30T08:44:58+01:00 INFO [vpn] starting
2023-05-30T08:44:58+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T08:44:58+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T08:44:58+01:00 INFO [wireguard] Connecting to 92.60.40.194:51820
2023-05-30T08:44:58+01:00 INFO [wireguard] Wireguard is up
2023-05-30T08:44:58+01:00 INFO [firewall] setting allowed input port 57160 through interface tun0...
2023-05-30T08:44:58+01:00 WARN [dns over tls] cannot update files: Get "https://www.internic.net/domain/named.root": dial tcp: lookup www.internic.net on 1.1.1.1:53: read udp 10.65.43.151:51584->1.1.1.1:53: i/o timeout
2023-05-30T08:44:58+01:00 INFO [dns over tls] attempting restart in 40s


3.33.0 logs:
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================

Running version v3.33.0 built on 2023-04-11T16:03:22.528Z (commit 16ecf48)

🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-05-30T08:33:48+01:00 INFO [routing] default route found: interface eth0, gateway 172.17.0.1 and assigned IP 172.17.0.4
2023-05-30T08:33:48+01:00 INFO [routing] local ethernet link found: eth0
2023-05-30T08:33:48+01:00 INFO [routing] local ipnet found: 172.17.0.0/16
2023-05-30T08:33:48+01:00 INFO [firewall] enabling...
2023-05-30T08:33:48+01:00 INFO [firewall] enabled successfully
2023-05-30T08:33:48+01:00 INFO [storage] creating /gluetun/servers.json with 13064 hardcoded servers
2023-05-30T08:33:49+01:00 INFO Alpine version: 3.17.3
2023-05-30T08:33:49+01:00 INFO OpenVPN 2.4 version: 2.4.12
2023-05-30T08:33:49+01:00 INFO OpenVPN 2.5 version: 2.5.8
2023-05-30T08:33:49+01:00 INFO Unbound version: 1.17.1
2023-05-30T08:33:49+01:00 INFO IPtables version: v1.8.8
2023-05-30T08:33:49+01:00 INFO Settings summary:
├── VPN settings:
|   ├── VPN provider settings:
|   |   ├── Name: mullvad
|   |   └── Server selection settings:
|   |       ├── VPN type: wireguard
|   |       ├── Cities: amsterdam
|   |       └── Wireguard selection settings:
|   └── Wireguard settings:
|       ├── Private key: KK...lU=
|       ├── Interface addresses:
|       |   └── redacted
|       └── Network interface: tun0
├── DNS settings:
|   ├── DNS server address to use: 127.0.0.1
|   ├── Keep existing nameserver(s): no
|   └── DNS over TLS settings:
|       ├── Enabled: yes
|       ├── Update period: every 24h0m0s
|       ├── Unbound settings:
|       |   ├── Authoritative servers:
|       |   |   └── cloudflare
|       |   ├── Caching: yes
|       |   ├── IPv6: no
|       |   ├── Verbosity level: 1
|       |   ├── Verbosity details level: 0
|       |   ├── Validation log level: 0
|       |   ├── System user: root
|       |   └── Allowed networks:
|       |       ├── 0.0.0.0/0
|       |       └── ::/0
|       └── DNS filtering settings:
|           ├── Block malicious: yes
|           ├── Block ads: no
|           ├── Block surveillance: no
|           └── Blocked IP networks:
|               ├── 127.0.0.1/8
|               ├── 10.0.0.0/8
|               ├── 172.16.0.0/12
|               ├── 192.168.0.0/16
|               ├── 169.254.0.0/16
|               ├── ::1/128
|               ├── fc00::/7
|               ├── fe80::/10
|               ├── ::ffff:7f00:1/104
|               ├── ::ffff:a00:0/104
|               ├── ::ffff:a9fe:0/112
|               ├── ::ffff:ac10:0/108
|               └── ::ffff:c0a8:0/112
├── Firewall settings:
|   ├── Enabled: yes
|   ├── VPN input ports:
|   |   └── 57160
|   └── Outbound subnets:
|       └── 192.168.1.0/24
├── Log settings:
|   └── Log level: INFO
├── Health settings:
|   ├── Server listening address: 127.0.0.1:9999
|   ├── Target address: cloudflare.com:443
|   ├── Read header timeout: 100ms
|   ├── Read timeout: 500ms
|   └── VPN wait durations:
|       ├── Initial duration: 6s
|       └── Additional duration: 5s
├── Shadowsocks server settings:
|   └── Enabled: no
├── HTTP proxy settings:
|   ├── Enabled: yes
|   ├── Listening address: :8888
|   ├── User: 
|   ├── Password: [not set]
|   ├── Stealth mode: no
|   ├── Log: no
|   ├── Read header timeout: 1s
|   └── Read timeout: 3s
├── Control server settings:
|   ├── Listening address: :8000
|   └── Logging: yes
├── OS Alpine settings:
|   ├── Process UID: 1001
|   ├── Process GID: 1001
|   └── Timezone: Europe/London
├── Public IP settings:
|   ├── Fetching: every 12h0m0s
|   └── IP file path: /tmp/gluetun/ip
└── Version settings:
    └── Enabled: yes
2023-05-30T08:33:49+01:00 INFO [routing] default route found: interface eth0, gateway 172.17.0.1 and assigned IP 172.17.0.4
2023-05-30T08:33:49+01:00 INFO [routing] adding route for 0.0.0.0/0
2023-05-30T08:33:49+01:00 INFO [firewall] setting allowed subnets...
2023-05-30T08:33:49+01:00 INFO [routing] default route found: interface eth0, gateway 172.17.0.1 and assigned IP 172.17.0.4
2023-05-30T08:33:49+01:00 INFO [routing] adding route for 192.168.1.0/24
2023-05-30T08:33:49+01:00 INFO [http server] http server listening on [::]:8000
2023-05-30T08:33:49+01:00 INFO [healthcheck] listening on 127.0.0.1:9999
2023-05-30T08:33:49+01:00 INFO [dns over tls] using plaintext DNS at address 1.1.1.1
2023-05-30T08:33:49+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T08:33:49+01:00 INFO [http proxy] listening on :8888
2023-05-30T08:33:49+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T08:33:49+01:00 INFO [wireguard] Connecting to 185.65.134.222:51820
2023-05-30T08:33:49+01:00 INFO [wireguard] Wireguard is up
2023-05-30T08:33:49+01:00 INFO [firewall] setting allowed input port 57160 through interface tun0...
2023-05-30T08:33:49+01:00 INFO [dns over tls] downloading DNS over TLS cryptographic files
2023-05-30T08:33:57+01:00 INFO [healthcheck] program has been unhealthy for 6s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-05-30T08:33:57+01:00 INFO [vpn] stopping
2023-05-30T08:33:57+01:00 INFO [firewall] removing allowed port 57160...
2023-05-30T08:33:57+01:00 ERROR [vpn] cannot get version information: Get "https://api.github.com/repos/qdm12/gluetun/releases": context canceled
2023-05-30T08:33:57+01:00 INFO [vpn] starting
2023-05-30T08:33:57+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T08:33:57+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T08:33:57+01:00 INFO [wireguard] Connecting to 193.32.249.66:51820
2023-05-30T08:33:57+01:00 INFO [wireguard] Wireguard is up
2023-05-30T08:33:57+01:00 INFO [firewall] setting allowed input port 57160 through interface tun0...
2023-05-30T08:33:57+01:00 INFO [healthcheck] healthy!
2023-05-30T08:33:59+01:00 WARN [dns over tls] cannot update files: Get "https://www.internic.net/domain/named.root": dial tcp: lookup www.internic.net on 1.1.1.1:53: read udp 10.65.43.151:45319->1.1.1.1:53: i/o timeout
2023-05-30T08:33:59+01:00 INFO [dns over tls] attempting restart in 10s
2023-05-30T08:34:09+01:00 INFO [dns over tls] downloading DNS over TLS cryptographic files
2023-05-30T08:34:10+01:00 INFO [dns over tls] downloading hostnames and IP block lists
2023-05-30T08:34:14+01:00 INFO [dns over tls] init module 0: validator
2023-05-30T08:34:14+01:00 INFO [dns over tls] init module 1: iterator
2023-05-30T08:34:14+01:00 INFO [dns over tls] start of service (unbound 1.17.1).
2023-05-30T08:34:14+01:00 INFO [dns over tls] generate keytag query _ta-4a5c-4f66. NULL IN
2023-05-30T08:34:14+01:00 INFO [dns over tls] generate keytag query _ta-4a5c-4f66. NULL IN
2023-05-30T08:34:14+01:00 INFO [dns over tls] ready
```

### Share your configuration

```yml
gluetun:
    image: qmcgaw/gluetun:latest
    container_name: gluetun
    network_mode: bridge
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    ports:
      - 8888:8888/tcp # HTTPProxy
    environment:
      - TZ=Europe/London
      - PUID=1001
      - PGID=1001
      - HTTPPROXY=on
      - VPN_SERVICE_PROVIDER=mullvad
      - VPN_TYPE=wireguard
      - WIREGUARD_PRIVATE_KEY=redacted
      - WIREGUARD_ADDRESSES=redacted
      - SERVER_CITIES=Amsterdam
      - FIREWALL_OUTBOUND_SUBNETS=192.168.1.0/24
      - FIREWALL_VPN_INPUT_PORTS=57160
    restart: always
```

**Comment by klarker** [comment]
exactly the same issue, reverted back to 3.34.1 and it´s now working. Ill wait for a fix before update.

**Comment by uncapped1599** [comment]
I am getting a similar restart issue with Mullvad but it is failing health check while attempting to add forwarded ports. Can confirm it started working when reverting back to v3.34.0:

`2023-05-30T10:20:11.913148647Z 2023-05-30T11:20:11+01:00 INFO [vpn] stopping
2023-05-30T10:20:11.913150259Z 2023-05-30T11:20:11+01:00 INFO [firewall] removing allowed port 56502...
2023-05-30T10:20:12.064926082Z 2023-05-30T11:20:12+01:00 INFO [vpn] starting
2023-05-30T10:20:12.064989024Z 2023-05-30T11:20:12+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T10:20:12.067882110Z 2023-05-30T11:20:12+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T10:20:12.068107099Z 2023-05-30T11:20:12+01:00 INFO [wireguard] Connecting to 141.98.252.222:51820
2023-05-30T10:20:12.068236332Z 2023-05-30T11:20:12+01:00 INFO [wireguard] Wireguard is up
2023-05-30T10:20:12.168855576Z 2023-05-30T11:20:12+01:00 INFO [firewall] setting allowed input port 56502 through interface tun0...
2023-05-30T10:20:48.169046115Z 2023-05-30T11:20:48+01:00 INFO [healthcheck] program has been unhealthy for 36s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-05-30T10:20:48.169071297Z 2023-05-30T11:20:48+01:00 INFO [vpn] stopping
2023-05-30T10:20:48.169072837Z 2023-05-30T11:20:48+01:00 INFO [firewall] removing allowed port 56502...
2023-05-30T10:20:48.297195494Z 2023-05-30T11:20:48+01:00 INFO [vpn] starting
2023-05-30T10:20:48.297215075Z 2023-05-30T11:20:48+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T10:20:48.300478121Z 2023-05-30T11:20:48+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T10:20:48.300778195Z 2023-05-30T11:20:48+01:00 INFO [wireguard] Connecting to 185.248.85.18:51820
2023-05-30T10:20:48.300931299Z 2023-05-30T11:20:48+01:00 INFO [wireguard] Wireguard is up
2023-05-30T10:20:48.401367362Z 2023-05-30T11:20:48+01:00 INFO [firewall] setting allowed input port 56502 through interface tun0...
2023-05-30T10:21:24.630182067Z 2023-05-30T11:21:24+01:00 INFO [dns over tls] downloading DNS over TLS cryptographic files
2023-05-30T10:21:31.413659720Z 2023-05-30T11:21:31+01:00 INFO [healthcheck] program has been unhealthy for 41s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
2023-05-30T10:21:31.413681686Z 2023-05-30T11:21:31+01:00 INFO [vpn] stopping
2023-05-30T10:21:31.413683294Z 2023-05-30T11:21:31+01:00 INFO [firewall] removing allowed port 56502...
2023-05-30T10:21:31.633194149Z 2023-05-30T11:21:31+01:00 INFO [vpn] starting
2023-05-30T10:21:31.633218710Z 2023-05-30T11:21:31+01:00 INFO [firewall] allowing VPN connection...
2023-05-30T10:21:31.636880088Z 2023-05-30T11:21:31+01:00 INFO [wireguard] Using available kernelspace implementation
2023-05-30T10:21:31.637053671Z 2023-05-30T11:21:31+01:00 INFO [wireguard] Connecting to 185.195.232.67:51820
2023-05-30T10:21:31.637197968Z 2023-05-30T11:21:31+01:00 INFO [wireguard] Wireguard is up
2023-05-30T10:21:31.737690442Z 2023-05-30T11:21:31+01:00 INFO [firewall] setting allowed input port 56502 through interface tun0...
2023-05-30T10:21:34.631934183Z 2023-05-30T11:21:34+01:00 WARN [dns over tls] cannot update files: Get "https://www.internic.net/domain/named.root": dial tcp: lookup www.internic.net on 1.1.1.1:53: read udp 10.64.70.97:59837->1.1.1.1:53: i/o timeout
2023-05-30T10:21:34.631967323Z 2023-05-30T11:21:34+01:00 INFO [dns over tls] attempting restart in 2m40s
2023-05-30T10:22:18.752795706Z 2023-05-30T11:22:18+01:00 INFO [healthcheck] program has been unhealthy for 46s: restarting VPN (see https://github.com/qdm12/gluetun/wiki/Healthcheck)
`

**Comment by capsel22** [comment]
in your docker-compose just specify the version number aka

qmcgaw/gluetun:v33.3.0

**Comment by uncapped1599** [comment]
here is the relevant bit of my docker compose for help on reverting to an older version:

`
version: "2.1"
services:
    gluetun:
        image: qmcgaw/gluetun:v3.34.0
        container_name: gluetun
        restart: unless-stopped
        cap_add:
          - NET_ADMIN
          - SYS_MODULE
        volumes:
            - ${CONFIG}/gluetun:/gluetun
            - /lib/modules:/lib/modules
`

**Comment by uncapped1599** [comment]
Here is an excerpt from docker compose to help reverting to an older version

```dockerfile
version: "2.1"
services:
    gluetun:
        image: qmcgaw/gluetun:v3.34.0
        container_name: gluetun
        restart: unless-stopped
        cap_add:
          - NET_ADMIN
          - SYS_MODULE
        volumes:
            - ${CONFIG}/gluetun:/gluetun
            - /lib/modules:/lib/modules
```

**Comment by qdm12** [comment]
@capsel22 

> Running version latest built on 2023-05-29T20:47:00.502Z (commit 4759392)

You are running the latest version, specify your image as `qmcgaw/gluetun:v3.34.2` (or with `v3` or `v3.34`, but then make sure you `docker-compose pull` to get the latest v3 release).

@klarker @uncapped1599 @capsel22 the latest image broke (shame on me) yesterday evening, I just pushed a fix in 2c30984a10bd0667a715fbd5f77b268a58a0af42 which should resolve all (at least most) environment variables issues.
